source/input.cpp

```diff
@@ -914,7 +917,7 @@ void SetDefaultButtonMap ()
 char* GetUSBControllerInfo()
 {
     static char info[70];
-    snprintf(info, 70, "Retrode: %s, XBOX360: %s, Hornet: %s", Retrode_Status(), XBOX360_Status(), Hornet_Status());
+    snprintf(info, 70, "Retrode: %s, XBOX360: %s, Hornet: %s, Mayflash: %s", Retrode_Status(), XBOX360_Status(), Hornet_Status(), Mayflash_Status());

```

Comment:
we might want to bump up this length?

Retrode: connected, XBOX360: connected, Hornet: connected, Mayflash: connected

would be 79 characters for example

Reply:
I've increased it to 100, which should be adequate